live-examples/html-examples/inline-text-semantics/rb.html

```diff
@@ -0,0 +1,5 @@
+<ruby>
+  <rb>上<rb>手
+  <rt>じよう<rt>ず
+  <rtc><rt>jou<rt>zu
```

Comment:
Nit: we use 4-space indent.

Reply:
I'm not sure about all this omitting of closing tags. It is correct I think, but it makes Code Mirror sad:

<img width="223" alt="screen shot 2018-07-11 at 5 20 36 pm" src="https://user-images.githubusercontent.com/432915/42605986-babfc076-852f-11e8-901c-0c727a2b7d44.png">

@schalkneethling , what do you think?

Reply:
Agree, please close all void, and non-void tags. Thanks!